geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/AlterAsyncEventQueueCommand.java

```diff
@@ -0,0 +1,167 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.management.internal.cli.commands;
+
+import static org.apache.geode.management.internal.cli.i18n.CliStrings.CREATE_ASYNC_EVENT_QUEUE__BATCHTIMEINTERVAL__HELP;
+import static org.apache.geode.management.internal.cli.i18n.CliStrings.CREATE_ASYNC_EVENT_QUEUE__MAXIMUM_QUEUE_MEMORY__HELP;
+import static org.apache.geode.management.internal.cli.i18n.CliStrings.IFEXISTS;
+import static org.apache.geode.management.internal.cli.i18n.CliStrings.IFEXISTS_HELP;
+
+import java.io.IOException;
+import java.util.Map;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.TransformerException;
+
+import org.springframework.shell.core.annotation.CliCommand;
+import org.springframework.shell.core.annotation.CliOption;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import org.apache.geode.distributed.internal.ClusterConfigurationService;
+import org.apache.geode.management.cli.CliMetaData;
+import org.apache.geode.management.cli.Result;
+import org.apache.geode.management.internal.cli.AbstractCliAroundInterceptor;
+import org.apache.geode.management.internal.cli.GfshParseResult;
+import org.apache.geode.management.internal.cli.exceptions.EntityNotFoundException;
+import org.apache.geode.management.internal.cli.i18n.CliStrings;
+import org.apache.geode.management.internal.cli.result.ResultBuilder;
+import org.apache.geode.management.internal.cli.result.TabularResultData;
+import org.apache.geode.management.internal.configuration.domain.Configuration;
+import org.apache.geode.management.internal.configuration.utils.XmlUtils;
+import org.apache.geode.management.internal.security.ResourceOperation;
+import org.apache.geode.security.ResourcePermission;
+
+/**
+ * this command currently only updates the cluster configuration. Requires server restart to pick up
+ * the changes.
+ */
+public class AlterAsyncEventQueueCommand implements GfshCommand {
+
+  static final String COMMAND_NAME = "alter async-event-queue";
+  static final String ID = "id";
+  static final String BATCH_SIZE = "batch-size";
+  static final String BATCH_TIME_INTERVAL = "batch-time-interval";
+  static final String MAX_QUEUE_MEMORY = "max-queue-memory";
+  static final String MAXIMUM_QUEUE_MEMORY = "maximum-queue-memory";
+
+  static final String COMMAND_HELP =
+      "alter attributes of async-event-queue, needs rolling restart for new attributes to take effect. ";
+  static final String ID_HELP = "Id of the async event queue to be changed.";
+  static final String BATCH_SIZE_HELP = CliStrings.CREATE_ASYNC_EVENT_QUEUE__BATCH_SIZE__HELP;
```

Comment:
For consistency, import this string from CliStrings as you did for the following help strings.